### DISCUSSION TITLE
CSV Converting to XLSX

### DISCUSSION THREAD

**Discussion by JCAvita** [CMSgov/hospital-price-transparency#103] [Q&A] [answered]
I know that this templet comes in CSV format. However, I would like to be able to convert it to a XLSX file instead. is these something that CMS would approve of?

**Comment by RidenourK** [comment, marked as answer]
@JCAvita. Thank you for your question. 

As indicated at [45 CFR 180.50(c)(2),](https://www.ecfr.gov/current/title-45/part-180#p-180.50(c)) Beginning July 1, 2024, the hospital's machine-readable file **must conform to a CMS template layout**, data specifications, and data dictionary for purposes of making public the standard charge information required under paragraph (b)(2) of this section.

CMS has made the CMS template available in three non-proprietary formats, CSV “tall”, CSV “wide”, and JSON, in the “[Documentation](https://github.com/CMSgov/hospital-price-transparency/tree/master/documentation)” section of this CMS Hospital Price Transparency - Data Dictionary GitHub repository.
